tests/components/light/test_intent.py

```diff
@@ -67,17 +85,21 @@
         "test",
         intent.INTENT_SET,
         {
-            "name": {"value": "Hello"},
-            "color": {"value": "blue"},
-            "brightness": {"value": "20"},
+            "name": {
+                "value": "Hello"
+            },
+            "color": {
+                "value": "blue"
+            },
+            "brightness": {
+                "value": "20"
+            },
         },
     )
     await hass.async_block_till_done()
 
-    assert (
-        result.speech["plain"]["speech"]
-        == "Changed hello 2 to the color blue and 20% brightness"
-    )
+    assert (result.speech["plain"]["speech"] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)